cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/frontends/cpp/CPPLanguage.kt

```diff
@@ -57,7 +57,7 @@ class CPPLanguage :
     @Transient
     override val simpleTypes =
         mapOf(
-            "bool" to BooleanType("bool", 1, this, NumericType.Modifier.SIGNED),
+            "bool" to BooleanType("bool", 1, this),
```

Comment:
If BooleanType is always size 1, we can probably make this the default

Reply:
It is the default, but the language argument comes the bitfield :D